dw-dialog.js

```diff
@@ -289,6 +289,14 @@ export class DwDialog extends LitElement {
     });
   }
 
+  connectedCallback() {
+    super.connectedCallback && super.connectedCallback();
+    if (window.visualViewport) {
+      window.visualViewport.addEventListener('scroll', this._viewportHandler.bind(this));
```

Comment:
As per our standard practice, `_viewportHandler` should be bound to `this` in Constructor.